src/containers/NewProject/NewProject.js

```diff
@@ -1,30 +1,127 @@
 import React from 'react';
-import { Button, Modal, Form, Input } from 'antd';
+import { Button, Modal, Form, Input, Steps, message, Menu, Dropdown, Icon, Card, Table, Divider, Tag } from 'antd';
```

Comment:
Other ant components being used (ie Table for 'checkout', Card for rooms display, and Steps for whole process)